src/Structs/Playlist.js

```diff
@@ -0,0 +1,9 @@
+import Track from './Track';
+
+export default class Playlist {
+  constructor(playlistObject, id) {
```

Comment:
Out of our control :+1: 

playlistObject is actually stored in another object and it's **key** is the ID.  It's strange store, but it's GPM's not ours :smile: